README.md

```diff
@@ -135,6 +135,23 @@ cd pre-assembly
 bundle install
 ```
 
+#### redis/resque
+
+The pre-assembly app uses resque backed by redis for job queing.  In order to run the tests or run the
```

Comment:
Should redis and reques be capitalized as proper names in this sentence and the next?

Reply:
updated